### PR TITLE
Stop parameterizing namespace in CI templates

### PR DESCRIPTION
**Pull request by 0xmichalis**
/cc stevekuznetsov

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,72 +16,72 @@ prow-crd:
...
 prow-services:
-	oc process -f cluster/ci/config/prow/openshift/deck.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/hook.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/horologium.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/jenkins_operator.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/jenkins_proxy.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/plank.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/sinker.yaml | oc apply -f -
-	oc process -f cluster/ci/config/prow/openshift/splice.yaml | oc apply -f -
+	oc process -f cluster/ci/config/prow/openshift/deck.yaml -p NAMESPACE=$(CI_NAMESPACE) | oc apply -f -
```
Comment (stevekuznetsov) [inline Makefile]: This is fishy. @csrwng do you have any opinions on it?

Comment (kargakis) [reply Makefile]: What do you mean?

Comment (stevekuznetsov) [reply Makefile]: I feel like having to specify the namespace on every object in a template is mis-use of a template

Comment (csrwng) [reply Makefile]: Looking at the templates, it seems that we could leave out the NAMESPACE parameter completely. @kargakis is there a specific case where the namespace must be specified? I think leaving it out of the template will make it more portable and easier to instantiate

Comment (kargakis) [reply Makefile]: The only reason is to have a preset default so we wont depend on the context. Agreed that it makes the templates less portable. I am going to remove it.

Comment (kargakis) [reply Makefile]: Another reason for parameterizing the namespace I just remembered is that we are using clusterroles and they require the namespace of the service account to be present inside a clusterrolebinding. We don't really need clusterroles (https://github.com/openshift/release/issues/125) so I went ahead and switched them to roles.

Comment (kargakis) [reply Makefile]: I want to use sinker for garbage collecting pods in the origin-ci namespace. Meaning sinker needs to use a clusterrole. How are we going to infer the namespace inside the template in that case?

Comment (csrwng) [reply Makefile]: so I've added cleanup to the pipelines for unit tests and verification tests ... but builds are not yet getting cleaned up. I think that can be done with the regular build pruner though.

**Comment by kargakis** [comment]
Changed clusterroles to roles inside the cluster.

**Pull request merged**